app/helpers/service_provider_helper.rb

```diff
@@ -72,9 +72,9 @@ def redirect_uri_list(service_provider = @service_provider)
   end
 
   def yamlized_sp(service_provider)
-    key_from_issuer = JSON.parse(service_provider.to_json).dig('production_issuer').presence ||
-                      service_provider.issuer
-    yamlable_json = { "'#{key_from_issuer}'" => config_hash(service_provider) }
+    key_from_issuer = service_provider.issuer
+    yamlable_json = { "#{key_from_issuer}": config_hash(service_provider) }
+    yamlable_json.deep_stringify_keys!
```

Comment:
It's definitely not critical, but I think this became necessary with the switch from string keys to symbol keys. Is it still needed if the keys are switched back to strings?

Reply:
Apologies, I might be misunderstanding, but my comment is only for the internal Ruby keys used in the hash rather than the keys or values in the resulting YAML.

Reply:
Yeah I think I misunderstood what you were saying (didn't see your response and deleted my comment because I was going to rewrite) . You are correct that we wouldn't need the `deep_stringify_keys` if we kept it as string keys. I don't have a super strong opinion about it besides it seems like the more popular stylistic choice to go with symbols.

Reply:
I don't feel particularly strongly, though in this case I lean towards string keys as they were previously string keys, and YAML returns string keys from parsing so consistency in that regard can help with interoperability back and forth.

Reply:
Gotcha, in that case I will revert the change!